tests/queries/0_stateless/2010_lc_native.python

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import socket
+import os
+
+CLICKHOUSE_HOST = os.environ.get('CLICKHOUSE_HOST', '127.0.0.1')
+CLICKHOUSE_PORT = int(os.environ.get('CLICKHOUSE_PORT_TCP', '900000'))
+CLICKHOUSE_DATABASE = os.environ.get('CLICKHOUSE_DATABASE', 'default')
+
+def writeVarUInt(x, ba):
+    for _ in range(0, 9):
+
+        byte = x & 0x7F
+        if x > 0x7F:
+            byte |= 0x80
+
+        ba.append(byte)
+
+        x >>= 7
+        if x == 0:
+            return
+
+
+def writeStringBinary(s, ba):
+    b = bytes(s, 'utf-8')
+    writeVarUInt(len(s), ba)
+    ba.extend(b)
+
+
+def readStrict(s, size = 1):
+    res = bytearray()
+    while size:
+        cur = s.recv(size)
+        # if not res:
+        #     raise "Socket is closed"
+        size -= len(cur)
+        res.extend(cur)
+
+    return res
+
+
+def readUInt(s, size=1):
+    res = readStrict(s, size)
+    val = 0
+    for i in range(len(res)):
+        val += res[i] << (i * 8)
+    return val
+
+def readUInt8(s):
+    return readUInt(s)
+
+def readUInt16(s):
+    return readUInt(s, 2)
+
+def readUInt32(s):
+    return readUInt(s, 4)
+
+def readUInt64(s):
+    return readUInt(s, 8)
+
+def readVarUInt(s):
+    x = 0
+    for i in range(9):
+        byte = readStrict(s)[0]
+        x |= (byte & 0x7F) << (7 * i)
+
+        if not byte & 0x80:
+            return x
+
+    return x
+
+
+def readStringBinary(s):
+    size = readVarUInt(s)
+    s = readStrict(s, size)
+    return s.decode('utf-8')
+
+
+def sendHello(s):
+    ba = bytearray()
+    writeVarUInt(0, ba) # Hello
```

Comment:
Maybe some code would be simpler using [struct](https://docs.python.org/3/library/struct.html) module, not sure